network/manager.go

```diff
@@ -115,8 +115,22 @@ func (nm *networkManager) Initialize(config *common.PluginConfig, isRehydrationR
 func (nm *networkManager) Uninitialize() {
 }
 
+// Remove the lock on the key-value store
+func (nm *networkManager) unlockStore() error {
+	if nm.store != nil {
+		err := nm.store.Unlock()
+		if err != nil {
+			log.Printf("[cni] Failed to unlock store: %v.", err)
+			return err
+		}
+	}
+	nm.store = nil
+	return nil
+}
+
 // Restore reads network manager state from persistent store.
 func (nm *networkManager) restore(isRehydrationRequired bool) error {
+	log.Printf("entering restore func...")
```

Comment:
can it be `[net] Entering restore function`

Reply:
I put this for my own debugging. Forgot to remove it. Will delete it in next commit. I need to fix the error checking for nm.unlockStore() too